openssl/src/tests.rs

```diff
@@ -217,7 +218,6 @@ fn tls_client_stream_test(server_addr: IpAddr, mtls: bool) {
 }
 
 #[allow(unused_variables)]
-#[cfg(feature = "tls")]
 fn config_mtls(
```

Comment:
btw, none of my mtls stuff is working at the moment. The OpenSSL library has an ugly habit of just grabbing STDIN and trying to read for passwds, etc. Never got to the bottom of it. It was before I had rustls that I started this though, so maybe when I fix this, I'll just support rustls. Issue #100

Reply:
No... I just wanted you to be aware in case any of the methods "appear" to be unused.